plugins/BEdita/API/src/Middleware/CorsMiddleware.php

```diff
@@ -229,8 +135,9 @@ protected function buildCors(ServerRequestInterface $request, ResponseInterface
         }
 
         $response = $corsBuilder->build();
-        if (!empty($origin) && !$response->hasHeader('Access-Control-Allow-Origin')) {
-            throw new ForbiddenException('Origin not allowed');
+
+        if ($response->getHeaderLine('Access-Control-Allow-Origin') != '*') {
```

Comment:
(minor) Here I'd prefer a strict check `!==`. Since `*` is not a string that could be type-casted to a strange value, it is probably pretty much the same, so it's just a matter of taste. :)

(Same at line 129)